app/src/main/cpp/skyline/services/fssrv/IFileSystemProxy.cpp

```diff
@@ -69,6 +69,18 @@ namespace skyline::service::fssrv {
         return {};
     }
 
+    Result IFileSystemProxy::OpenDataStorageByDataId(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        auto storageId{request.Pop<StorageId>()};
```

Comment:
Split this into seperate commit, and add another commit with the ngword stuff

Reply:
well this is "the" ngword stuff, unless you mean the romfs file with the list

Reply:
ye

Reply:
(The file itself is fine to add)